Katas/RomanNumerals/RomanNumeralConverter.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace RomanNumerals
+{
+    public class RomanNumeralConverter
+    {
+        public RomanNumeralConverter() {
+        }
+
+        private Tuple<int, string>[] digits = {
```

Comment:
I would often argue against the use of a tuple in c#. Some languages handle them very well but I don't think c# is one of those languages

Why not use a dictionary or another alternative that c# has cleaner syntax for?